database/database_virtual_accounts.go

```diff
@@ -7,11 +7,30 @@ import (
 )
 
 func GetBalanceVA(vaNum string, db *sql.DB) (va models.VirtualAccounts, err error) {
-	var balanceFloat float32
-	err = db.QueryRow("SELECT va_balance FROM virtual_accounts WHERE va_num = ($1) ", vaNum).Scan(&balanceFloat)
-	if err != nil {
-		return
+	err = db.QueryRow("SELECT va_balance FROM virtual_accounts WHERE va_num = ($1) ", vaNum).Scan(&va.VaBalance)
+	return
+}
```

Comment:
I think it's better to change the return type to integer instead of the whole struct

Reply:
changed.

Reply:
okay